pandas/tests/indexes/datetimes/test_ops.py

```diff
@@ -593,6 +594,12 @@ def test_nat_new(self):
         exp = np.array([tslib.iNaT] * 5, dtype=np.int64)
         tm.assert_numpy_array_equal(result, exp)
 
```

Comment:
there are very likely tests that are raising ValueError that should now be NullFrequencyError, pls change them.

You can find them very easily, by temporarly changing the error where NullFrequencyInhertis to something else (e.g. KeyError) and then seeing what raises

Reply:
Will do.

Reply:
Found and changed one.

Reply:
ok great